index.test-d.ts

```diff
@@ -25,6 +25,7 @@ expectType<typeof ky>(ky.create({}));
 expectType<typeof ky>(ky.extend({}));
 expectType<HTTPError>(new HTTPError(new Response));
 expectType<TimeoutError>(new TimeoutError);
+expectType<symbol>(stop);
```

Comment:
I'm not sure if it matters, but shouldn't this be `Symbol`?

Reply:
No, it’s a primitive, so should be lowercased (both work, at least until I get XO working with ambient TS files).